packages/commonwealth/client/scripts/helpers/feature-flags.ts

```diff
@@ -23,3 +27,14 @@ export const featureFlags = {
 };
 
 export type AvailableFeatureFlag = keyof typeof featureFlags;
+
+const unleashConfig = {
+  url: process.env.UNLEASH_FRONTEND_SERVER_URL,
+  clientKey: process.env.UNLEASH_FRONTEND_API_TOKEN,
+  refreshInterval: 120,
+  appName: 'commonwealth-web',
+};
+
+export const openFeatureProvider = process.env.UNLEASH_FRONTEND_API_TOKEN
```

Comment:
this conditional does not make sense if `UNLEASH_FRONTEND_API_TOKEN` has a default value in `webpack.base.config.js`. I think we should not use default value there. Otherwise, local provider does not work